File: Bluetooth_Arduino/bluetoothProject/app/src/main/java/com/example/bluetoothproject/SplashActivity.java
package com.example.bluetoothproject;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {
        @Override
        protected void onCreate(Bundle savedInstanceState) {
                super.onCreate(savedInstanceState);
                setContentView(R.layout.activity_main);

                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
                startActivity(intent);
        }
}

File: PlantEngine_TermProject/app/src/main/java/com/example/plantengine_termproject/SplashActivity.kt
package com.example.plantengine_termproject

import android.content.Intent
import android.os.Bundle
import android.os.SystemClock
import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        SystemClock.sleep(1000)
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}